packages/http/src/mocker/index.ts

```diff
@@ -58,7 +58,7 @@ const mock: IPrismComponents<IHttpOperation, IHttpRequest, IHttpResponse, IHttpM
   config,
 }) => {
   const payloadGenerator: PayloadGenerator = config.dynamic
-    ? partial(generate, resource, resource['__bundle__'])
+    ? partial(generate, resource, resource['__bundled__'])
```

Comment:
Why this has changed ?? how it was working previously?

Reply:
Dereferencing was properly working with previous code (resource['__bundle__']) But getting issue in the API response. So due that I make change here.